apps/server/src/modules/room/room-api.module.ts

```diff
@@ -1,13 +1,14 @@
 import { AuthorizationModule } from '@modules/authorization';
 import { Module } from '@nestjs/common';
 import { LoggerModule } from '@src/core/logger';
+import { BoardModule } from '../board';
 import { RoomMemberModule } from '../room-member/room-member.module';
+import { UserModule } from '../user';
 import { RoomController, RoomUc } from './api';
 import { RoomModule } from './room.module';
-import { UserModule } from '../user';
 
 @Module({
-	imports: [RoomModule, AuthorizationModule, LoggerModule, RoomMemberModule, UserModule],
+	imports: [RoomModule, AuthorizationModule, LoggerModule, RoomMemberModule, BoardModule, UserModule],
```

Comment:
I don't think that room should import board here.
We will have a much easier time if we can prevent that.